recirq/algorithmic_benchmark_library.py

```diff
@@ -83,6 +83,12 @@ class BenchmarkConfig:
         spec_class=TiltedSquareLatticeLoschmidtSpec,
         executable_generator_func=get_all_tilted_square_lattice_executables,
         configs=[
+            BenchmarkConfig(
+                short_name='small-v1',
```

Comment:
Is there a good place to put a synopsis about what each benchmark is/does?  Either as a line comment or as a structured field?  Once we have a bunch of these, it's going to be more difficult to keep track of what each of them does.

Reply:
good idea, added